pkg/api/scylla/v1/types_cluster.go

```diff
@@ -375,6 +392,14 @@ type RepairTaskSpec struct {
 
 	// host specifies a host to repair. If empty, all hosts are repaired.
 	Host *string `json:"host,omitempty" mapstructure:"host,omitempty"`
+
+	// ignoreDownHosts specifies that the manager should not attempt to repair the nodes that are down, i.e. in status DN.
+	// +optional
+	IgnoreDownHosts bool `json:"IgnoreDownHosts,omitempty" mapstructure:"ignore_down_hosts,omitempty"`
+
+	// tags specifies an extra set of tags describing the task.
+	// +optional
+	Tags []string `json:"tags,omitempty"`
 }
```

Comment:
> This is not repair-specific field

The api on our side doesn't really match SM's, so there isn't really a better place for it - the shared inline struct is `SchedulerTaskSpec`, which doesn't fit the case here. The fields in RepairTaskSpec/BackupTaskSpec seem to be a mix of mapped properties and some shared fields, some even taken out of the `Schedule`on SM side. tl;dr; it's a mess that we have to live with now for backwards compatibility

> Do you have a use case for this field?

Nope, I just didn't want to leave it out while synchronising all the other fields.

> first validate its existence on SM side.

Well, it's in the model https://github.com/scylladb/scylla-manager/commit/5a73edfd5b2c5b344829ddd58adec641d965d245

Reply:
I would scope this down to the fields we see a meaning for. We can always add a field but never to remove one. So let's avoid adding them just because they are present in manager API. I don't see a big reason for `IgnoreDownHosts` either at this point.

Reply:
Removed the tags. As for `IgnoreDownHosts`, I think a reasonable rule of thumb is to allow setting whatever sctool allows for.

Reply:
@tnozicka after internal conversation I removed the other fields, only leaving `cron` and `timezone` related to it